.github/workflows/lint-telemetry-definitions.yaml

```
@@ -0,0 +1,26 @@
+name: Lint Telemetry Definitions
```

could be just "Lint", all linting can be in one CI job
```suggestion
name: Lint
```

---

I'd prefer to parallelize all of the linting, having each job focus on its own domain. We only have the one thing being linted in CI, and it's easy to change course though. Let's leave it as-is for now.